src/Rule/Url.php

```diff
@@ -20,10 +20,12 @@
 /**
  * Validates that the value is a valid HTTP or HTTPS URL.
```

Comment:
In my opinion the current description is a bit confusing, because the rule does not perform the actual validation / checking - it's done by handler.